src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,357 +1,4105 @@
     url: https://opensource.org/licenses/MIT
+  title: X-Road Central Server Admin API
+  version: "1.0.0"
 servers:
-  - url: /api/v1
-    description: basepath for API v1
+  - description: basepath for API v1
+    url: /api/v1
+tags:
+  - name: backups
+    description: resources related to Backups and Backup Restoration
+  - name: central-services
+    description: resources related to Central Services
+  - name: certification-services
+    description: resources related to Certification Services
+  - name: certification-services-ocsp-responders
+    description: resources related to OCSP Responders belonging to Certification Services
+  - name: certification-services-intermediate-cas
+    description: resources related to Intermediate Certificate Authorities belonging to Certification Services
+  - name: globalconf
+    description: general resources related to Global Configuration
+  - name: globalconf-anchors
+    description: resources related to Global Configuration anchors
+  - name: globalconf-tokens
+    description: resources related to Global Configuration tokens
+  - name: globalconf-keys
+    description: resources related to Global Configuration keys
+  - name: global-groups
+    description: resources related to Globalgroups
+  - name: initialization
+    description: resources related to Central Server initialization
+  - name: intermediate-cas
+    description: resources related to Intermediate Certificate Authorities
+  - name: member-classes
+    description: resources related to Central Server's member classes
+  - name: ocsp-responders
+    description: resources related to OCSP Responders
+  - name: security-servers
+    description: resources related to Security Servers
+  - name: security-servers-certs
+    description: resources related to Security Servers' Authentication Certificates
+  - name: system
+    description: resources related to System info
+  - name: timestamping-services
+    description: resources related to Central Server's Timestamping Services
 paths:
-  /system/version:
+  /backups:
     get:
-      tags:
-        - system
-      summary: get information for the system version
-      operationId: systemVersion
+      description: <h3>Administrator views the backups for the central server.</h3>
+      operationId: getBackups
       responses:
         '200':
-          description: system version information
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Version'
+                description: array of backup objects
+                items:
+                  $ref: '#/components/schemas/Backup'
+                type: array
+                uniqueItems: true
+          description: list of central server backups
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
```

Comment:
Yes, it was intentional. 404 cannot happen here, since the endpoint always returns a list (empty or not)

I don't see 400 happening because the resource path is plain `/backups` and has no parameters (https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1). Maybe considering the whole path `/api/v1/backups` there could be `/api/v2/backups`, or similar error, but not sure if that counts as 400 either 🤔

Perhaps 406 should have been left in the list because the headers of the request might be off.

Reply:
All right, thanks. Was just wondering if there was some kind of bigger change hidden here. And true, 400 is probably not possible after all...